services/movie_session.py

```diff
@@ -42,3 +42,14 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling the case where the movie session with the given `session_id` does not exist before attempting to delete it. You can use `get_object_or_404` or handle the `MovieSession.DoesNotExist` exception.